fe1-web/src/core/components/Input.tsx

```diff
@@ -45,7 +46,7 @@ const Input = (props: IPropTypes) => {
   return (
     <View style={inputStyleSheet.container}>
       <TextInput
-        style={inputStyles}
+        style={[{ fontFamily: customFont } as TextStyle, inputStyles]}
         placeholderTextColor={Color.inactive}
         editable={enabled || false}
```

Comment:
It depends on the context. 
For example, if you use the digital cache module, then you will use the autocomplete list with a dropdown choice. It is easier to choose from this menu if all tokens have the same size. In this case, the font of the input should be monospaced for coherence. 

And since it is an optional prop, it does not change anything for other usages of others input fields.